docs/pages/bdd-references/given.md

```diff
@@ -11,14 +11,17 @@ parent: BDD Reference
 `GIVEN` directive is use to set the initial standard about the type of the entity that you are searching for.
 Every Scenario must have a `GIVEN` as the first step defining the context.
 
+If the scenario starts with `Given I have aws_s3_bucket defined`, the step below will have all the aws_s3_bucket resources defined in its context.
+
+
 Few examples can be ;
 
 ```gherkin
 Given I have AWS S3 Bucket defined
 ```
 
 ```gherkin
-Given I have aws provider configured
+Given I have aws_s3_bucket defined
 ```
```

Comment:
Removed this since it's misleading: `Given I have aws provider configured` won't capture all aws providers. It will look for a provider declaration in terraform file.